cclient/serialization/json/tests/get_neighbors.c

```diff
@@ -26,28 +26,45 @@ void test_get_neighbors_serialize_response(void) {
   serializer_t serializer;
   init_json_serializer(&serializer);
   char_buffer_t* out = char_buffer_new();
+
   char const * TEST_JSON_TEXT =
       "{\"neighbors\":[{"
-      "\"address\":\"" TEST_NEIGHBORS_NEIGHBOR1
-      "\",\"numberOfAllTransactions\":" STR(TEST_NEIGHBORS_NUMALLTX1)
-      ",\"numberOfInvalidTransactions\":" STR(TEST_NEIGHBORS_NUMINVALIDTX1)
-      ",\"numberOfNewTransactions\":" STR(TEST_NEIGHBORS_NUMNEWTX1)
-      "},{\"address\":\"" TEST_NEIGHBORS_NEIGHBOR2
-      "\",\"numberOfAllTransactions\":" STR(TEST_NEIGHBORS_NUMALLTX2)
-      ",\"numberOfInvalidTransactions\":" STR(TEST_NEIGHBORS_NUMINVALIDTX2)
-      ",\"numberOfNewTransactions\":" STR(TEST_NEIGHBORS_NUMNEWTX2) "}]}";
+      "\"address\":\"" TEST_NEIGHBORS_NEIGHBOR1 "\""
+      ",\"numberOfAllTransactions\":" STR(TEST_NEIGHBORS_NUMALLTX1)
```

Comment:
These JSON elements are number, not string, please reference the API document. 
https://docs.iota.org/docs/iri/0.1/references/api-reference#getneighbors

Reply:
I think `STR` is just a macro to concatenate, but the result is not enclosed in `"` so it will be a number in the JSON

Reply:
Sorry, I overlooked the quotation marks. :P